service/routes.py

```diff
@@ -157,7 +173,40 @@ def update_shopcarts(shopcart_id):
 ######################################################################
 # UPDATE A SHOPCART ITEM
 ######################################################################
+@app.route("/shopcarts/<int:shopcart_id>/items/<int:item_id>", methods=["PUT"])
+def update_shopcart_item(shopcart_id, item_id):
+    """
+    Update a Shopcart Item
+
+    This endpoint will update an Shopcart Item based the body that is posted
+    """
+    app.logger.info("Request to update shopcart item with id: %s", item_id)
+    check_content_type("application/json")
+    import time
+    start_time=time.time()
+    # See if the shopcart exists and abort if it doesn't
+    shopcart = Shopcart.find(shopcart_id)
+    if not shopcart:
+        abort(
```

Comment:
Use error(status_code, reason) helper method instead of calling abort to enable logging.